boa/src/builtins/object/mod.rs

```diff
@@ -130,6 +133,7 @@ impl Object {
             internal_slots: FxHashMap::default(),
             properties: FxHashMap::default(),
             symbol_properties: FxHashMap::default(),
+            prototype: Value::null(),
```

Comment:
Hmmm I see many fields here have the default value. Can't we use the `..Self::default()` notation?

Reply:
We can use the ..Self::default() notation but, eventually we will have to remove them they are not spec compliant.